app/jobs/bulk_update/add_trainees/import_rows_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module BulkUpdate
+  module AddTrainees
+    class ImportRowsJob < ApplicationJob
+      queue_as :bulk_update
+
+      def perform(id:)
```

Comment:
If we pass the object this would have to change

```suggestion
      def perform(trainee_upload)
```